Configurations/unix-Makefile.tmpl

```diff
@@ -346,23 +346,23 @@ test: tests
 	@ : {- output_off() if $disabled{tests}; "" -}
 	( cd test; \
 	  mkdir -p test-runs; \
-	  SRCTOP=../$(SRCDIR) \
-	  BLDTOP=../$(BLDDIR) \
+	  SRCTOP="../$(SRCDIR)" \
+	  BLDTOP="../$(BLDDIR)" \
 	  RESULT_D=test-runs \
 	  PERL="$(PERL)" \
 	  EXE_EXT={- platform->binext() -} \
-	  OPENSSL_ENGINES=`cd ../$(BLDDIR)/engines 2>/dev/null && pwd` \
-	  OPENSSL_MODULES=`cd ../$(BLDDIR)/providers 2>/dev/null && pwd` \
+	  OPENSSL_ENGINES="`cd "../$(BLDDIR)/engines" 2>/dev/null && pwd`" \
+	  OPENSSL_MODULES="`cd "../$(BLDDIR)/providers" 2>/dev/null && pwd`" \
 	  OPENSSL_DEBUG_MEMORY=on \
-	    $(PERL) ../$(SRCDIR)/test/run_tests.pl $(TESTS) )
+	    $(PERL) "../$(SRCDIR)/test/run_tests.pl" $(TESTS) )
 	@ : {- if ($disabled{tests}) { output_on(); } else { output_off(); } "" -}
 	@echo "Tests are not supported with your chosen Configure options"
 	@ : {- output_on() if !$disabled{tests}; "" -}
 
 list-tests:
 	@ : {- output_off() if $disabled{tests}; "" -}
 	@SRCTOP="$(SRCDIR)" \
-	 $(PERL) $(SRCDIR)/test/run_tests.pl list
+	  $(PERL) "$(SRCDIR)/test/run_tests.pl" list
```

Comment:
intentional to change the indentation here?

Reply:
Yes. Indentation isn't all that consistent throughout the whole file, but this case was an exception even when compared to the surrounding code (see the `test` rule above this one), where 2 spaces were used in an almost identical multi-line command. So I synced this line with that one.

Reply:
ah, okay